src/Controller/CreationController.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace App\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+
+class CreationController extends AbstractController
```

Comment:
Which creation? creation is an action, while controllers are named after the resource they're controlling